plugins/cpython.py

```diff
@@ -0,0 +1,98 @@
+from plugins.api import Plugin
+from incparser.annotation import Annotation, Footnote, Heatmap, ToolTip
+
+import os
+import os.path
+import subprocess
+import tempfile
+
+class CPythonFuncProfileMsg(Annotation):
+    def __init__(self, annotation):
+        self._hints = [ToolTip(), Footnote()]
+        super(CPythonFuncProfileMsg, self).__init__(annotation)
+
+    def get_hints(self):
+        return self._hints
+
+
+class CPythonFuncProfileVal(Annotation):
+    def __init__(self, annotation):
+        self._hints = [Heatmap()]
+        super(CPythonFuncProfileVal, self).__init__(annotation)
+
+    def get_hints(self):
+        return self._hints
+
+
+class CPythonProfiler(Plugin):
+    """CPython 2 prfofiler.
+    Requires the Python standard library.
+    """
+
+    def __init__(self):
+        super(CPythonProfiler, self).__init__("Python 2.7.5",
+                                              "CPython profiler")
+
+    def run_tool(self):
+        f = tempfile.mkstemp()
+        self.export(f[1])
+
+        # python -m cProfile [-o output_file] [-s sort_order] myscript.py
+        proc = subprocess.Popen(["python2",
```

Comment:
The language name needs to match the language in the "languages" list that Eco holds internally. At the moment the entry for Python is "Python 2.7.5", regardless of the Python 2 version installed on your machine. I agree that is a little strange, but it is documented in the README.

I pushed a fix to prevent the README from being exec'd this morning BTW,